README.md

```diff
@@ -170,6 +178,21 @@ Do not hesitate to contact
 need help with the integration of your PDE solver.
 
 
+External Matrix Equation Solvers
+--------------------------------
+
+pyMOR also integrates matrix equation solvers for system-theoretic methods.
+The following solver libraries are included:
```

Comment:
This is not entirely true. pyMOR only provides bindings (in `pymor.bindings`) for Py-M.E.S.S. and Slycot, and those need to be installed separately. Maybe mention https://pymor.readthedocs.io/en/latest/getting_started.html#installation?